src/Game/Games/CookieClicker/Controler/ServerCookieController.java

```diff
@@ -105,5 +105,4 @@ public void update(Observable o, Object arg) {
             ServerCookieController.this.play((ActionEnum) messageDataObject.getData());
```

Comment:
verify the type of the message just in case

Reply:
I was thinking you could test the value returned by `messageDataObject.getType()` instead

Reply:
The both are correct but I'll send a commit to use the type instead of the value